webpack/assets/javascripts/react_app/components/Layout/Navigation.js

```diff
@@ -16,7 +16,7 @@ import {
 } from '../../Root/Context/ForemanContext';
 
 const titleWithIcon = (title, iconClass) => (
-  <div>
+  <div className="ellipsis">
```

Comment:
ellipsis behavior comes from jquery which we would like to avoid. I think it will be best to split the name into multiple lines if its too long, with css

Reply:
Do you mean something like this?
![image](https://github.com/theforeman/foreman/assets/78563507/c6cf5392-48bf-43e7-babc-a953ffab96fa)

Reply:
Yes :)